.forge-snapshots/BinPoolSwapTest#testGas_exactOutputMultipleBin_WithEmptyBins.snap

```diff
@@ -1 +1 @@
-109014
\ No newline at end of file
+137102
```

Comment:
can see it cost 30k more gas when there's 3 empty bins (eg. add-remove liquidity from bin however bin id still in TreeMath due to minShare)

Reply:
Yeah, and once a bin get "polluted", it will never get back 😂

What if we add a "symmetric operation" i.e. the last ppl removing liquidity receives MINIUM_SHARE ?

Reply:
per discussed, we go with https://github.com/pancakeswap/pancake-v4-core/pull/215 over https://github.com/pancakeswap/pancake-v4-core/pull/217 as #215 is safer